tools/dockerfile/grpc_artifact_centos6_x64/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Copyright 2016 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Docker file for building gRPC artifacts.
+
+##################
+# Base
+
+FROM dockcross/manylinux2010-x64
```

Comment:
what's the difference between this image and the manylinux2010 ones we are using for building python artifacts?
Ideally we'd like to use as few distinct external images as possible (we don't have control of them so it has potential to break us).

Reply:
`dockcross` adds more features on top of `manylinux2010` such as cmake which is required to build .NET artifacts.